ci/docs-builder.py

```diff
@@ -67,16 +76,8 @@ def main():
 
     builder.ensure_dir(WORKING_DIR, clean=True)
```

Comment:
This is also in the script and is currently redundant. Maybe it will be disincluded when the script is made import side-effect free.